src/pages/validator/amplifier/integrator-onboarding.mdx

```diff
@@ -0,0 +1,87 @@
+# Integrate your chain
+
+## Prerequisites
+
+- Make sure you have `wasm` enabled.
+- You should know how to deploy and interact with a CosmWasm contract. Refer to the [Osmosis CosmWasm testnet deployment tutorial](https://docs.osmosis.zone/cosmwasm/testnet/cosmwasm-deployment/) if you’d like a refresher — just replace `osmosisid` with `axelard` and use the `arm64` version of the [contract optimizer](https://github.com/CosmWasm/optimizer) if you are working on a [Mac with Apple silicon](https://support.apple.com/en-us/116943). Run `--gas auto --gas-adjustment 2` to provide your transactions with sufficient gas.
+
+## Devnet information
+
+- Devnet RPC: `http://devnet.rpc.axelar.dev:26657`
+- Devnet LCD (REST API/JSON RPC): `http://devnet.rpc.axelar.dev:1317`
```

Comment:
get the right RPC/LCD from the devnet config (see the end of the config)

Reply:
they are different in the config
https://github.com/axelarnetwork/axelar-contract-deployments/commit/e6f549de873e0ad211a073218f172d03f6e92c3b#diff-367dc4b25cf481a8731c0ddb11b1ac629986cbb6f10381303be600f7fd267490R502

Reply:
Ah, I see. Corrected, thanks for pointing that out.